src/components/imageDownloader/imageDownloader.js

```diff
@@ -295,6 +300,12 @@ import template from './imageDownloader.template.html';
             reloadBrowsableImages(page, apiClient);
         });
 
+        page.querySelector('#chkShowParentImages').addEventListener('change', function () {
+            browsableImageStartIndex = 0;
```

Comment:
SonarQube picked up that there is a lot of duplication in these event handlers. It would be best to factor the common parts out to a single function.